src/main/java/com/aws/greengrass/util/platforms/windows/WindowsPlatform.java

```diff
@@ -73,6 +73,7 @@
 import static com.sun.jna.platform.win32.WinNT.WRITE_OWNER;
 
 public class WindowsPlatform extends Platform {
+    public static final String LOADER_LOGS_FILE_NAME = "greengrass.out.log";
```

Comment:
why greengrass.out instead of loader?

Reply:
That's the default behavior for a Windows service that we have no control over.